src/components/organisms/AdminVenueView/components/Spaces/Spaces.tsx

```diff
@@ -28,7 +29,12 @@ interface SpacesProps {
 export const Spaces: React.FC<SpacesProps> = ({ venue: space }) => (
   <AdminPanel variant="bound" className="Spaces">
     <AdminSidebar>
-      <SpaceEditForm space={space} />
+      {/* @debt TODO: use userId and worldSlug here */}
+      <SpaceEditForm
+        space={space}
+        userId={"" as UserId}
+        worldSlug={"" as WorldSlug}
+      />
```

Comment:
[NOTE] This component could be made accepting optional props or just not accept them at all and rely on `useWorldAndSpaceByParams()` hook inside it. That way it will be more easily moved and embedded in other components.